resources/assets/uad_lists.json

```diff
@@ -17529,7 +17529,7 @@
     "dependencies": null,
     "neededBy": null,
     "labels": null,
-    "removal": "Advanced"
+    "removal": "Recommended"
```

Comment:
As I mentioned before; a backend package that never executes on its own. Might be worth being a bit more careful with these.

Reply:
> As I mentioned before; a backend package that never executes on its own
Are you sure about that? The manifest has the `RECEIVE_BOOT_COMPLETED` permission.

https://beta.pithus.org/report/1cd9df93333bb4e4cd083a610e4bb556bf180dc9ea93d4cd9861e6163cb9f0b1